### PR TITLE
Allow for more fluid .modal-body class

### PR DESCRIPTION
**Pull request by N6REJ**
Pull Request for Issue #22331 22331 .

### Summary of Changes
removed max-height limits for .modal-body content.

### Testing Instructions
go to template manager and click on beez 3 template preview image.
Using the element inspector change height of the image to 500px;
Contents will overflow...
show-overflow

### Expected result
now modal resizes to keep contents visible.
image

### Actual result
show-overflow

### Documentation Changes Required

ios js fix for max-height may need to be adjusted as well.

### CODE REVIEW

File: administrator/templates/isis/css/template-rtl.css [hunk 1]
```diff
@@ -3775,7 +3775,7 @@ input[type="submit"].btn.btn-mini {
 .modal-body {
 	width: 98%;
 	position: relative;
-	max-height: 400px;
+	display: content;s
```
Comment (ciar4n) [inline administrator/templates/isis/css/template-rtl.css]: Typo

**Comment by ciar4n** [comment]
There is some compatibility issues using `display: contents`... https://caniuse.com/#feat=css-display-contents

**Comment by brianteeman** [comment]
There are also a lot of unrelated files being committed here that should not be

File: administrator/templates/isis/css/template-rtl.css (hunk 1 above)
Comment (N6REJ) [reply administrator/templates/isis/css/template-rtl.css]: crap.. i'll fix it right now.

**Comment by N6REJ** [comment]
> There are also a lot of unrelated files being committed here that should not be

where?  There are 6 .css files involved.
EDIT: WTH!... i'll try to fix that too.  Just fyi, idky those files are there they were never touched.

**Comment by N6REJ** [comment]
> There is some compatibility issues using `display: contents`... https://caniuse.com/#feat=css-display-contents

I'll look into that.

**Comment by N6REJ** [comment]
@brianteeman here's the changes I pushed.
[modal-body.txt](https://github.com/joomla/joomla-cms/files/2408910/modal-body.txt)

**Comment by brianteeman** [comment]
They didnt magically get added

**Comment by brianteeman** [comment]
Those may be the changes you made but the other files are in your branch which is why they show up here

**Comment by N6REJ** [comment]
@brianteeman I don't dispute that but they don't show in the committ's in netbeans nor do I know how to remove them.

**Comment by N6REJ** [comment]
closing this to redo the pr.

**Pull request closed without merging**